src/openzaak/components/autorisaties/tests/admin/test_autorisaties_admin.py

```diff
@@ -548,52 +548,6 @@ def test_changes_send_notifications(self, mock_notif):
             }
         )
 
-    @override_settings(
-        NOTIFICATIONS_DISABLED=False,
-        OPENZAAK_DOMAIN="openzaak.example.com",
-        OPENZAAK_REWRITE_HOST=True,
-        ALLOWED_HOSTS=["testserver", "openzaak.example.com"],
-    )
-    @patch("notifications_api_common.viewsets.send_notification.delay")
-    def test_changes_send_notifications_with_openzaak_domain_setting(self, mock_notif):
```

Comment:
Please restore this test. I think it catches a bug here

Reply:
I fixed the behavior for the authorizations admin notifications and restored the test